cpp/tests/binaryop/binop-compiled-fixed_point-test.cpp

```diff
@@ -33,14 +33,14 @@
 namespace cudf::test::binop {
 
 template <typename T>
-struct FixedPointCompiledTestBothReps : public cudf::test::BaseFixture {
+struct FixedPointCompiledTestAllReps : public cudf::test::BaseFixture {
```

Comment:
I think we can just go down to ` FixedPointCompiledTest` since we see the test suite is using `cudf::test::FixedPointTypes`

Reply:
Yea that is a good point. Also prevents churn if we ever get 256bit decimal types :)